Sources/Testing/Testing.docc/exit-testing.md

```diff
@@ -107,11 +107,12 @@ func `Customer won't eat food unless it's nutritious`(_ food: Food) async {
 }
 ```
 
-- Note: If you use this macro with a Swift compiler version lower than 6.3, it
-  doesn't support capturing state.
+- Note: If you use the ``expect(processExitsWith:observing:_:sourceLocation:performing:)``
```

Comment:
`require` too.

Reply:
Ah I missed that because the original text had "this macro" and there were only two to choose from, of which `require` wasn't one.